packages/extension/src/ui/features/accounts/multisig/CreateMultisigScreen/MultisigSecondStep.tsx

```diff
@@ -0,0 +1,114 @@
+import { FieldError, H1 } from "@argent/ui"
+import { icons } from "@argent/ui"
+import { Box, Button, Center, Flex } from "@chakra-ui/react"
+import { Controller, useFormContext } from "react-hook-form"
+
+import { isEmptyValue } from "../../../../../shared/utils/object"
+import { useAppState } from "../../../../app.state"
+import { useNetwork } from "../../../networks/useNetworks"
+import { useSignerKey } from "../useSignerKey"
+import { ScreenLayout } from "./ScreenLayout"
+import { useCreateMultisig } from "./useCreateMultisig"
+import { FieldValues } from "./useCreateMultisigForm"
+
+const { AddIcon, MinusIcon } = icons
+
+export const MultisigSecondStep = ({
+  index,
+  goBack,
+  goNext,
+}: {
+  index: number
+  goBack: () => void
+  goNext: () => void
+}) => {
+  const { switcherNetworkId } = useAppState()
+  const { encodedPubKey } = useSignerKey()
+  const { createMultisigAccount, isError } = useCreateMultisig()
+  const network = useNetwork(switcherNetworkId)
+  const {
+    control,
+    formState: { errors },
+    getValues,
+  } = useFormContext<FieldValues>()
+
+  const handleCreateMultisig = async () => {
+    if (isEmptyValue(errors) && encodedPubKey) {
+      const result = await createMultisigAccount({
+        networkId: network.id,
+        signers: [...getValues("signerKeys").map((i) => i.key), encodedPubKey],
+        threshold: getValues("confirmations").toString(),
+      })
+      if (result) {
+        goNext()
+      }
+    }
+  }
+
+  return (
+    <ScreenLayout
+      subtitle="How many owners must confirm each transaction before it’s sent?"
+      currentIndex={index}
+      title="Set confirmations"
+      goBack={goBack}
+      back={true}
+    >
+      <Box my="2" width="100%">
+        <Controller
+          name="confirmations"
+          control={control}
+          defaultValue={1}
+          rules={{ required: true }}
+          render={({ field }) => (
+            <>
+              <Center>
+                <Flex direction="column" width="100%">
+                  <Flex
+                    justifyContent="space-between"
+                    width="100%"
+                    p="3"
+                    backgroundColor="neutrals.800"
+                    borderRadius={2}
```

Comment:
```suggestion
                    borderRadius={'lg'}
```

radius use different tokens; we are still using defaults here https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/theme/src/foundations/radius.ts